### PR TITLE
Migrate instructions from image.sc

### PR DESCRIPTION
**Pull request by joshmoore**
see: https://forum.image.sc/t/omero-and-ldap-questions/38917/2

cc: @sbesson @pwalczysko

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,60 @@ The initial password can be changed in [`docker-compose.yml`](docker-compose.yml
...
+version: 1
+dn: uid=t,ou=Users,dc=openmicroscopy,dc=org
+changetype: modify
+replace: givenName
+givenName: X.
+```
+
+## Testing with omero-ldaptool
+
+[omero-ldap](https://github.com/glencoesoftware/omero-ldaptool) provides a way to check if your
```
Comment (sbesson) [inline README.md]:
```suggestion
[omero-ldaptool](https://github.com/glencoesoftware/omero-ldaptool) provides a way to check if your
```

File: README.md [hunk 2]
```diff
@@ -20,3 +20,60 @@ The initial password can be changed in [`docker-compose.yml`](docker-compose.yml
...
+Then from the `omeroserver` container, download the configuration:
+
+```
+docker-compose exec omeroserver /opt/omero/server/venv3/bin/omero config get --show-password > cfg
+```
+
+Then, run the build and the tool:
+
+```
+./gradlew installDist
```
Comment (sbesson) [inline README.md:66]: This raises the question of whether the tool could be pre-built into the server image.

**Review by sbesson — APPROVED** [review]
One minor suggestion. Otherwise :+1: for extending the README for better usability.

File: README.md (hunk 2 above)
Comment (joshmoore) [reply README.md:66]: or be its own image?

**Review by sbesson — APPROVED** [review]

**Pull request merged**